src/common/hotbar-types.ts

```diff
@@ -33,14 +33,18 @@ export interface HotbarItem {
   }
 }
 
-export type Hotbar = Required<HotbarCreateOptions>;
+export type Hotbar = Required<CreateHotbarData>;
 
-export interface HotbarCreateOptions {
+export interface CreateHotbarData {
   id?: string;
   name: string;
   items?: Tuple<HotbarItem | null, typeof defaultHotbarCells>;
```

Comment:
Not your change, but do you think we could do without this `Tuple`? The same should be doable with more simplicity.

Reply:
Well we only want 12 items in Hotbar so I think encoding that into the type system should be good.